schemas/com.growthbook/experiment_viewed/jsonschema/1-0-0

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for an GrowthBook experiment viewed event",
+  "self": {
+    "vendor": "com.growthbook",
+    "name": "experiment_viewed",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "experimentId": {
```

Comment:
It looks like there's [a lot of information](https://github.com/growthbook/growthbook/blob/main/packages/sdk-js/src/types/growthbook.ts#L170) available in `Experiment` and `Result`, but nothing called `experimentId`. Should it be `experimentKey` to better map to `experiment.key`? Could we get a `description` for each field in the schema so it's easier to know what should be implemented?

Reply:
Calling this field "experimentId" is consistent with our other trackers, and is how we expect the column to be named (or aliased to) for the experiment report. 

I added descriptions to all the fields.